modules/geth/charts.go

```diff
@@ -0,0 +1,143 @@
+package geth
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dims   = module.Dims
+	Dim    = module.Dim
+)
+
+var charts = Charts{
+	chartAncientChainData.Copy(),
+	chartChaidataDisk.Copy(),
+	chartTxPoolPending.Copy(),
+	chartBlockProcessingTime.Copy(),
+	chartTxPoolQueued.Copy(),
+	chartP2PNetwork.Copy(),
+	chartP2PNetworkDetails.Copy(),
+	chartNumberOfPeers.Copy(),
+	chartRpcInformation.Copy(),
+}
+
+var (
+
+	chartAncientChainData = Chart{
+		ID:    "chaindata_ancient",
+		Title: "Ancient Chaindata",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.chaindata",
+		Dims: Dims{
+			{ID:ethDbChainDataAncientRread , Name: "Ancient chaindata reads"},
+			{ID: ethDbChainDataAncientWrite, Name: "Ancient chaindata writes", Mul: -1},
```

Comment:
```suggestion
		Ctx:   "geth.eth_db_chaindata_ancient_io",
		Dims: Dims{
			{ID:ethDbChainDataAncientRread , Name: "reads"},
			{ID: ethDbChainDataAncientWrite, Name: "writes", Mul: -1},
```

I see this metrics type is `gauge`, but

<details>
<summary>I see these metrics type is gauge, but</summary>

```cmd
# TYPE eth_db_chaindata_ancient_read gauge
eth_db_chaindata_ancient_read 47512691

# TYPE eth_db_chaindata_ancient_write gauge
eth_db_chaindata_ancient_write 700013496
```

</details>

we need to ensure that:

 - it is not `counter`, the value looks like cumulative value and not actual sample value.
 - it is bytes, but not operations.